### ISSUE TITLE
[Question]  build_dataset error log: 'sequence-labeling-model is not in the custom_datasets registry group named-entity-recognition. Please make sure the correct version of ModelScope library is used

### ISSUE THREAD

**Issue by tangjielong928** [modelscope/AdaSeq#44]
### What is your question?

在使用adaseq框架做NER任务时报错：
build_dataset error log: 'sequence-labeling-model is not in the custom_datasets registry group named-entity-recognition. Please make sure the correct version of ModelScope library is used

### What have you tried?

原来是用的modelscope 1.7.1后面尝试升级到1.9.5依然不行，运行到这个地方就卡住了。

完整报错信息：
2024-06-20 10:39:37,335 - modelscope - INFO - PyTorch version 1.10.1+cu111 Found.
2024-06-20 10:39:37,336 - modelscope - INFO - Loading ast index from /home/tangjielong/.cache/modelscope/ast_indexer
2024-06-20 10:39:37,365 - modelscope - INFO - Loading done! Current index file version is 1.7.1, with md5 55cda22e675324e12989be11cd8d8653 and a total number of 861 components indexed
2024-06-20 10:39:38,305 - modelscope - WARNING - The reference has been Deprecated in modelscope v1.4.0+, please use from modelscope.msdatasets.dataset_cls.custom_datasets import TorchCustomDataset
2024-06-20 10:39:38,401 - INFO - adaseq.data.dataset_manager - Will use a custom loading script: /data0/tangjielong/MNER_LLM/AdaSeq-master/adaseq/data/dataset_builders/named_entity_recognition_dataset_builder.py
Downloading and preparing dataset named_entity_recognition_dataset_builder/default to /data0/tangjielong/.cache/huggingface/datasets/named_entity_recognition_dataset_builder/default-df0a7beb617cd5ee/0.0.0/db737b9bb893f20fb03d04403a30bf7c033256c212b7e9f0ebc6e9c958535c51...
Downloading data: 216kB [00:00, 1.40MB/s]
Dataset named_entity_recognition_dataset_builder downloaded and prepared to /data0/tangjielong/.cache/huggingface/datasets/named_entity_recognition_dataset_builder/default-df0a7beb617cd5ee/0.0.0/db737b9bb893f20fb03d04403a30bf7c033256c212b7e9f0ebc6e9c958535c51. Subsequent calls will reuse this data.
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:00<00:00, 829.68it/s]
2024-06-20 10:39:40,242 - INFO - adaseq.data.dataset_manager - First sample in train set: {'id': '0', 'tokens': ['New', 'Post', ':', 'Blackburn', 'Festival', 'of', 'Voice', '2017'], 'spans': [{'start': 3, 'end': 7, 'type': 'MISC'}], 'mask': [True, True, True, True, True, True, True, True]}
2024-06-20 10:39:40,643 - INFO - adaseq.data.preprocessors.sequence_labeling_preprocessor - label_to_id: {'O': 0, 'B-LOC': 1, 'I-LOC': 2, 'E-LOC': 3, 'S-LOC': 4, 'B-MISC': 5, 'I-MISC': 6, 'E-MISC': 7, 'S-MISC': 8, 'B-ORG': 9, 'I-ORG': 10, 'E-ORG': 11, 'S-ORG': 12, 'B-PER': 13, 'I-PER': 14, 'E-PER': 15, 'S-PER': 16}
Some weights of the model checkpoint at xlm-roberta-large were not used when initializing XLMRobertaModel: ['lm_head.layer_norm.weight', 'lm_head.layer_norm.bias', 'lm_head.dense.bias', 'lm_head.dense.weight', 'lm_head.bias']

This IS expected if you are initializing XLMRobertaModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
This IS NOT expected if you are initializing XLMRobertaModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
** build_dataset error log: 'sequence-labeling-model is not in the custom_datasets registry group named-entity-recognition. Please make sure the correct version of ModelScope library is used.'
** build_dataset error log: 'sequence-labeling-model is not in the custom_datasets registry group named-entity-recognition. Please make sure the correct version of ModelScope library is used.'

### What's your environment?

- ModelScope Version: 1.7.1 and 1.9.5
- PyTorch Version:  1.10.1
- transformers: 4.30.2
- Python version: 3.7

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tangjielong928** [comment]
I have solved it.

**Comment by zhoufanjiang** [comment]
> 我已经解决了。

你好我也遇到了同样的报错，请问这里是怎么解决的呢 @tangjielong928

**Comment by tangjielong928** [comment]
> > 我已经解决了。
> 
> 你好我也遇到了同样的报错，请问这里是怎么解决的呢 @tangjielong928

这个不是报错好像实在加载模型，我是等待一段时间就好了 @zhoufanjiang
